package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "symphony-integration-jira",
+  "version": "0.11.0-SNAPSHOT",
+  "description": "Symphony App Integrations - Universal Webhook",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "npm test -- --watch",
+    "webpack": "./node_modules/webpack/bin/webpack.js -p --config ./webpack.config.prod.js",
+    "watch": "webpack-dev-server --progress --colors --hot --inline --https",
+    "license-check": "node-license-validator --allow-licenses WTFPL Apache Apache-2 BSD-like BSD BSD-2-Clause BSD-3-Clause Apache-2.0 MIT ISC Unlicense MIT/X11 \"MIT / http://rem.mit-license.org\" \"Public Domain\" --allow-packages domelementtype@1.3.0 domhandler@2.3.0 domutils@1.5.1 extsprintf@1.0.2 ripemd160@0.2.0 rx-lite@3.1.2 shellwords@0.1.0 stream-cache@0.0.2 verror@1.3.6"
+  },
+  "jest": {
+    "transform": {
+      ".*": "./node_modules/babel-jest"
+    },
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "./src/test/webapp/__mocks__/fileMock.js",
+      "\\.(css|less)$": "identity-obj-proxy"
+    }
+  },
+  "license": "Apache-2.0",
+  "author": "Symphony",
+  "devDependencies": {
+    "babel-core": "^6.14.0",
+    "babel-jest": "^18.0.0",
+    "babel-loader": "^6.2.5",
+    "babel-plugin-transform-object-rest-spread": "^6.22.0",
+    "babel-preset-es2015": "^6.14.0",
+    "babel-preset-react": "^6.11.1",
+    "babel-preset-stage-0": "^6.22.0",
+    "copy-webpack-plugin": "^4.0.1",
+    "css-loader": "^0.23.1",
+    "enzyme": "^2.7.1",
+    "eslint": "^3.7.1",
+    "eslint-config-airbnb": "^12.0.0",
+    "eslint-config-google": "^0.6.0",
+    "eslint-loader": "^1.5.0",
+    "eslint-plugin-import": "^1.16.0",
+    "eslint-plugin-jsx-a11y": "^2.2.2",
+    "eslint-plugin-react": "^6.3.0",
+    "file-loader": "^0.9.0",
+    "gulp": "^3.9.1",
+    "html-webpack-plugin": "^2.16.0",
+    "identity-obj-proxy": "^3.0.0",
+    "img-loader": "^1.3.1",
+    "jest": "^18.1.0",
+    "less": "^2.7.1",
+    "less-loader": "^2.2.3",
+    "react-addons-test-utils": "^15.4.2",
+    "style-loader": "^0.13.0",
+    "url-loader": "^0.5.7",
+    "webpack": "^1.12.14",
+    "webpack-dev-server": "^1.14.1",
+    "webpack-stream": "^3.2.0"
+  },
+  "dependencies": {
+    "babel-polyfill": "^6.23.0",
+    "react": "^15.3.2",
+    "react-dom": "^15.4.2",
+    "react-redux": "^5.0.2",
+    "react-router": "^3.0.2",
+    "redux": "^3.6.0",
+    "redux-saga": "^0.14.3",
+    "symphony-integration-commons": "file:///home/pdarde/symphony/app/App-Integrations-FE-Commons"
```

Comment:
Today we have the version "0.11.0-SNAPSHOT", as well as Jira. Which should be the correct version?